src/components/PopupWithConfirm.js

```diff
@@ -0,0 +1,36 @@
+import Popup from "../components/Popup";
+
+class PopupWithConfirm extends Popup {
+  constructor({ popupSelector,}) {
+    super({ popupSelector});
+    this._submitButton = this._popupElement.querySelector(".modal__button");
+    this._submitButtonContent = this._submitButton.textContent;
+  }
+
+
+        setSubmitAction(action) {
+            this._handleSubmitCallback = action;
+          }
+        
+          setEventListeners() {
+            this._popupElement.addEventListener("submit", (evt) => {
+              evt.preventDefault();
+              this._handleSubmitCallback();
+            });
+        
+            super.setEventListeners();
+          }
+
+          setLoading(isLoading, loadingText = "Saving..."){
```

Comment:
`"Saving..."`

In `PopupWithConfirm`, there should be `Deleting...` if you delete a card

Reply:
My remark hasn't been corrected https://github.com/UffyLane/se_project_aroundtheus/pull/1#discussion_r2060647786